packages/typespec-test/test/multiclient_modular/generated/typespec-ts/src/index.ts

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+export { Resource, CustomPage } from "./api/foo/models.js";
+export {
+  CreateOrUpdateOptions,
+  GetOptions,
+  DeleteOptions,
+  ListOptions,
+} from "./api/foo/operations.js";
+export { FooClient } from "./FooClient.js";
+export { ClientOptions, RequestOptions } from "./common/interfaces.js";
+export { Resource, CustomPage } from "./api/bar/models.js";
```

Comment:
In this case, because both subclients has used shared models in the typespec, and we export both models.ts,  whch is duplicated.  I wonder what is the right thing to do here ?  should we generate a common models.ts or should we try to rename one of them ?

Reply:
If we can identify common models it would be great just to generate and export them once